draft-ietf-ppm-dap.md

```diff
@@ -2236,7 +2236,7 @@ shares recovered during the aggregation sub-protocol.
 ##### Size Check
 
 The query configuration specifies the minimum batch size, `min_batch_size`. The
-Aggregator checks that `len(X) >= min_batch_size`, where `X` is the set of
+Aggregator checks that `len(X) >= min_batch_size`, where `X` is the set of valid
```

Comment:
I considered throwing in a reference to either `{{aggregate-flow}}` or `{{validating-inputs}}` to clarify just what we mean by "valid", but I think it's clear enough: the document overall discusses report validity all over the place.

Reply:
We should do the same under `{#fixed-size-batch-validation}` as well below.